### DISCUSSION TITLE
How To Use RNF's `useLoader` In Next.js?

### DISCUSSION THREAD

**Discussion by fweth** [vercel/next.js#35540] [Help]
Hi, I want to use `useLoader` from `@react-three/fiber` in Next.js but get the error `Error: ReactDOMServer does not yet support Suspense`.  (Apparently `Suspense` is needed for `useLoader`, I imported it from `react`.)  I found [this](https://nextjs.org/docs/advanced-features/react-18/overview) document and tried to install `next@latest`, `react@rc` and `react-dom@rc` on top of my document with no success, the new error read `Cannot set properties of undefined (setting 'reactRoot')`.  Then I tried to create a new project with `npx create-next-app@latest`, but that didn't work either.  My code is very basic and can be seen [here](https://codesandbox.io/s/loader-vqrjvx) as a React app.  Any ideas how to bring `useLoader` to Next.js?

**Comment by Theantipioneer** [comment]
anyone found an answer?
